crates/uv/src/commands/project/sync.rs

```diff
@@ -74,6 +75,12 @@ pub(crate) async fn sync(
         InstallTarget::from(&project)
     };
 
+    if project.workspace().pyproject_toml().has_scripts()
+        && !project.workspace().pyproject_toml().is_package()
+    {
+        warn_user!("Skipping scripts installation because this project is not a package. To install them, consider setting `tool.uv.package = true` or configuring a custom `build-system`.");
```

Comment:
I'd say:

> Skipping installation of entry points (`project.scripts`) because this project is not packaged; to install entry points, set `tool.uv.package = true` or define a `build-system`

This is relatively nuanced, so just a few notes:

- We want to say "entry points" because that's the [official term](https://packaging.python.org/en/latest/specifications/entry-points/) and just "scripts" could be confused with some `foo.py` script in the project.
- We don't use a trailing period in our CLI messages
- Usually you're just requesting use of a common build-system, "custom" doesn't feel like quite the right fit
- "packaged" vs "is a package" to match the language in the relevant [docs](https://docs.astral.sh/uv/concepts/projects/#configuring-project-packaging)